skabase/SKABaseDevice/SKABaseDevice/SKABaseDevice.py

```diff
@@ -416,17 +416,17 @@ def write_testMode(self, value):
     dtype_out='str', 
     )
     @DebugIt()
```

Comment:
All the decorators that aren't commented out will also cause confusion, and probably end up being applied to the next method, `GetVersionInfo` - not sure what will happen with that.

Reply:
Done. Commit f51ec80 addresses this.